docs/building-your-quest/conditions/k8s-backend-config-var-defined.md

```diff
@@ -0,0 +1,42 @@
+---
+title: k8s_backend_config_var_defined
```

Comment:
it's just backend var, users shouldn't care that we're using k8s

Reply:
They do care though. They use the k8s resource and it should be clear that these config vars are k8s related.

Reply:
Decided to leave as is, since we would like to make users' life easier by guiding them to search k8s for deployment related actions